tests_e2e/tests/agent_publish/agent_publish.py

```diff
@@ -79,6 +80,14 @@ def get_ignore_errors_before_timestamp(self) -> datetime:
         timestamp = self._ssh_client.run_command("agent_publish-get_agent_log_record_timestamp.py")
         return datetime.strptime(timestamp.strip(), u'%Y-%m-%d %H:%M:%S.%f').replace(tzinfo=UTC)
```

Comment:
```suggestion
        return datetime.strptime(timestamp.strip(), u'%Y-%m-%d %H:%M:%S.%f%z').replace(tzinfo=UTC)
```

the script returns a timestamp which is datetime aware. This format string needs to include the timezone info